Criteo forks/develop (#2729)
segmentio/segment-docs: src/connections/destinations/catalog/actions-criteo-audiences/index.md

---
title: Criteo Audiences (Actions) Destination
hide-personas-partial: true
hide-boilerplate: false
hide-dossier: true
id:
---
{% include content/plan-grid.md name="actions" %}

Criteo Audiences (Actions) enables advertisers to send Segment Persona Audiences to Criteo using Criteo's Audience API.

> By using Segment's Persona Audiences with Criteo, you have the opportunity to get traffic and drive conversions from new and existing customers as well as re-engaging existing customers with hyper-relevant ads that promote product discovery.

Author: @naevma would you be able to advise?

> By using Segment's Persona Audiences with Criteo, you have the opportunity to get traffic and drive conversions from new and existing customers as well as re-engaging existing customers with hyper-relevant ads that promote product discovery.

Reviewer: @stayseesong - You can update this to increase as you suggested. I made some changes and here's my commit. https://github.com/criteo-forks/segment-docs/commit/1589e46c4c83874754e2cf02dff06b1c8f03cdae
How do we add this to this PR? please advise.

> By using Segment's Persona Audiences with Criteo, you have the opportunity to get traffic and drive conversions from new and existing customers as well as re-engaging existing customers with hyper-relevant ads that promote product discovery.

Reviewer 2: Hi @pharika31, thanks for the updates!

Can you push to this PR? We had to recreate the original Criteo PR in our repository because we could not push edits to the original.

> By using Segment's Persona Audiences with Criteo, you have the opportunity to get traffic and drive conversions from new and existing customers as well as re-engaging existing customers with hyper-relevant ads that promote product discovery.

Reviewer: Hi @markzegarelli, this is a new branch on your Segment docs repository. How do I push from Criteo forks to this PR? can you please help with the workflow? not sure why you couldn't push from original PR.

> By using Segment's Persona Audiences with Criteo, you have the opportunity to get traffic and drive conversions from new and existing customers as well as re-engaging existing customers with hyper-relevant ads that promote product discovery.

Reviewer: @markzegarelli - Sounds good. This commit should have all the changes based on your review comments: https://github.com/criteo-forks/segment-docs/commit/1589e46c4c83874754e2cf02dff06b1c8f03cdae Let us know if this works!

## Benefits of Criteo Audiences (Actions)

> This integration creates a direct connection between Segment and Criteo for a higher match rate of email identifiers. Other benefits include:   

Reviewer 3: ```suggestion
Benefits of the Critero Audiences (Actions) destination include:
- **Improved email matching**: This integration creates a direct connection between Segment and Criteo for a higher match rate of email identifiers. 

```

> - **Fewer settings**: Unlike Criteo's Filter destination, this destination doesn't require any custom copy and paste code. You only need the Advertiser ID and the Criteo API credentials.

Reviewer 3: Custom or copy and paste? They seem to be two separate ideas to me (custom code seems like it would be written directly for someone, while copy and paste seems more like a one-size-fits-all solution.) Is there a UI that allows you to input information about your site and then someone could copy and paste an end result?

Also, would it be your Advertiser ID?
```suggestion
- **Fewer settings**: Unlike Criteo's Filter destination, this destination doesn't require any copy and paste code. You only need your Advertiser ID and Criteo API credentials.
```

Author: @naevma could you please advise?

Reviewer 2: @stayseesong Looking at the context here, I think `the Advertiser ID` is correct, since this identifier comes from the end-user's devices.

---
{% include content/plan-grid.md name="actions" %}

Criteo Audiences (Actions) enables advertisers to send Segment Persona Audiences to Criteo using Criteo's Audience API.

By using Segment's Persona Audiences with Criteo, you have the opportunity to get traffic and drive conversions from new and existing customers as well as re-engaging existing customers with hyper-relevant ads that promote product discovery.

## Benefits of Criteo Audiences (Actions)

This integration creates a direct connection between Segment and Criteo for a higher match rate of email identifiers. Other benefits include:   

- **Fewer settings**: Unlike Criteo's Filter destination, this destination doesn't require any custom copy and paste code. You only need the Advertiser ID and the Criteo API credentials.

> - **Criteo Audience**: You don't need a Criteo audience ID as the audience gets created on the fly using the Personas audience name. This enables audience Segment names to be consistent across Segment and Criteo.

Reviewer 3: Does the first instance of segment in the last sentence need to be capitalized? The sentence reads to me like it's referring primarily to audience segmentation!

```suggestion
- **Criteo Audience**: You don't need a Criteo audience ID as the audience gets created on the fly using the Personas audience name. This enables the names of audience segments to be consistent across Segment and Criteo.
```

Author: @naevma could you please clarify?

Reviewer 2: @stayseesong Isabella's edit is correct here.

- **Batching events and support for large audiences**: This destination supports batching which enables Criteo to receive large audiences without discrepancies.

- **Near real time audience updates**: Actions destination helps Criteo receive real-time events and add or remove users from Criteo audiences appropriately.

## Getting Started

### Prerequisites:

> Before connecting to the Criteo Audiences (Actions) destination, you must create a Criteo API Marketing Solutions app to generate your app credentials (client ID and secret) as per the guidelines in [Criteo's Developer's Portal](https://developers.criteo.com/marketing-solutions/docs/onboarding-checklist).  

Reviewer 3: ```suggestion
Before connecting to the Criteo Audiences (Actions) destination, you must create a Criteo API Marketing Solutions app to generate your app credentials (client ID and secret) as per the guidelines in [Criteo's Developer's Portal](https://developers.criteo.com/marketing-solutions/docs/onboarding-checklist){:target="_blank"}.
```

You will also need your Criteo Advertiser ID. Please reach out to your Criteo Account Strategist to get this ID.

### To Add the Criteo Audiences Destination:

1. From your Segment workspace, go to **Connections > Catalog** and click **Destinations**.

2. Search for **Criteo Audiences** in the Destinations Catalog and select the destination.

3. Click **Configure Criteo Audiences**.

4. Choose your **Personas** space for the Source as this destination only supports sending Personas Audiences to Criteo.

> 5. On the **Settings** tab, name your destination. For example, Criteo audiences – <audience name>.

Reviewer 3: Not sure if this is showing up on your localhost, but the `<audience name>` wasn't displaying when I built this locally: I had to escape the angle brackets to get this example to display correctly.

```suggestion
On the **Settings** tab, name your destination. For example, Criteo audiences – &lt;audience name&gt;.
```

6. Enter your Criteo Advertiser ID, API client ID and client secret.

7. Click **Save Changes**.  

8. In the **Mappings** tab, click **New Mapping** and select **Add Users to Audience**. Don't change any defaults.

9. Under the **Configure actions fields**, set **Enable Batching** to *Yes* and click **Save**.  

7. Repeat steps 8 and 9 for **Remove Users from Audience**.

8. **Enable** both mappings.

9. Go to the **Settings** tab and **Enable** the destination.

10. Go to your **Personas** space and click the **Audiences** tab. Select the source audience that you want to send to your Criteo Audiences destination.

> 11. Click **Add Destinations** and select the Criteo Audience destination you created. On the settings that appear on the right-hand side, toggle the **Send Track** option on and disable **Send Identify**. Click **Save**.  

Reviewer 3: ```suggestion
11. Click **Add Destinations** and select the Criteo Audience destination you created. In the settings that appear on the right-hand side, toggle the **Send Track** option on and disable **Send Identify**. Click **Save**. 
```
